draft-thomson-quic-tls.md

```diff
@@ -602,6 +626,10 @@ ISSUE:
   service exposure to on-path attackers.  The only possible problem is in
   authenticating the initial value, so that peers can be sure that they haven't
   missed an initial message.
+  TODO: I have no objection to using a randomized starting packet number. Would
+  that be super valuable? It's not clear to me how important protecting against
+  off-path vs on-path is? We have talked about doing random packet number jumps
+  to replace per-packet entropy, so this is definitely plausible.
```

Comment:
The entropy in the connection ID might be enough that this isn't an issue.  I don't know for sure.